src/mudlet-lua/lua/Other.lua

```diff
@@ -184,8 +184,8 @@ function appendScript(name, luaCode, pos)
   pos = pos or 1
   assert(type(name) == "string", "appendScript: bad argument #1 type (script name as string expected, got "..type(name).."!)")
   assert(type(name) == "string", "appendScript: bad argument #2 type (lua code as string expected, got "..type(luaCode).."!)")
```

Comment:
:exclamation: This looks suspect (not saying it is anything you have done here) - this line can never be executed as it is testing the same condition as the previous line - I suspect it should have been:
```lua
    assert(type(luaCode) == "string", "appendScript: bad argument #2 type (lua code as string expected, got "..type(luaCode).."!)")
```

Reply:
Ah, you introduced it in #3519  .

Reply:
Good catch.